pyproject.toml

```diff
@@ -35,10 +36,18 @@ pytz = "2021.3"
 pyvista = {version = "^0.37.0", optional = true, extras = ["vtk"]}
 
 # see https://github.com/orgs/python-poetry/discussions/7937
-urllib3 = "<2" 
+urllib3 = "<2"
 geopandas = "^0.13.2"
 numpy = "<1.25"
 nzshm-model = {version = "^0.4.0"}
+mkdocs = "^1.6.0"
```

Comment:
should these be dev dependencies so that users don't need such a heavy installation?